gqcp/tests/Operator/SecondQuantized/SQTwoElectronOperator_test.cpp

```diff
@@ -47,10 +49,23 @@ BOOST_AUTO_TEST_CASE ( SQTwoElectronOperator_constructor ) {
 BOOST_AUTO_TEST_CASE ( SQTwoElectronOperator_zero_constructor ) {
 
     const size_t dim = 2;
-    GQCP::ScalarSQOneElectronOperator<double> op {dim};
+    GQCP::ScalarSQTwoElectronOperator<double> op {dim};
+
+    // Create a reference zero tensor
+    GQCP::QCRankFourTensor<double> ref (dim);
+
+    for (size_t i = 0; i < dim; i++) {
+        for (size_t j = 0; j < dim; j++) {
+            for (size_t k = 0; k < dim; k++) {
+                for (size_t l = 0; l < dim; l++) {
+                    ref(i,j,k,l) = 0;
+                }
+            }
+        }
+    }
 
     BOOST_CHECK_EQUAL(op.dimension(), dim);
-    BOOST_CHECK(op.parameters().isZero(1.0e-08));
+    BOOST_CHECK(op.parameters().isApprox(ref, 1.0e-08));
```

Comment:
I'm not sure why these lines have changed. Only the constructing line was wrong.

Reply:
As I've said in another comment as well, I changed this because the QCFourRankTensor does not have a method "isZero()". I'd be happy to write this differently if there's a more efficient method that you know of.

Reply:
Ah, you are absolutely correct! My bad.

However, you could use `ref.setZero()` to remove the `for`-loops.